packages/fuel-indexer-macros/src/schema.rs

```diff
@@ -295,8 +334,23 @@ fn process_type_def(
                             }
                         }
                     }
+
+                    impl From<#strct> for Json {
+                        fn from(value: #strct) -> Self {
+                            let s = serde_json::to_string(&value).expect("Serde error.");
+                            Self(s)
+                        }
+                    }
+
+                    impl From<Json> for #strct {
+                        fn from(value: Json) -> Self {
+                            let s: #strct = serde_json::from_str(&value.0).expect("Serde error.");
+                            s
+                        }
+                    }
                 })
             } else {
+                // TOOD: derive Default here
```

Comment:
Is there a reason why we're not adding `Default` to the derived traits right now?

Reply:
https://github.com/FuelLabs/fuels-rs/pull/977